authorization/src/main/java/uk/gov/di/ipv/cri/common/api/service/SessionRequestService.java

```diff
@@ -0,0 +1,121 @@
+package uk.gov.di.ipv.cri.common.api.service;
```

Comment:
remove this file for now
In the next PR you can introduce an AuthorisationRequestValidator